okteto-pipeline.yml

```diff
@@ -1,5 +1,6 @@
 deploy:
-  - helm upgrade --install rabbitmq okteto/rabbitmq
+  - helm repo add okteto https://apps.okteto.com
```

Comment:
After this line, you need to run `helm repo update`. It's going to work without it because the installer job already does that, but you will need it for `okteto deploy`